data-processing/tests/test_extract_definitions.py

```diff
@@ -13,7 +13,7 @@
 
 @pytest.fixture(scope="module")
 def model():
-    model = DefinitionDetectionModel(['TERM-DEF', 'ABBR-EXP', 'SYM-NICK'])
+    model = DefinitionDetectionModel(['AI2020', 'DocDef2', 'W00'])
```

Comment:
What does it mean for the tasks to have the names 'AI2020', 'DocDef2' and 'W00', when the tasks are named 'TERM-DEF', 'ABBR-EXP', and 'SYM-NICK' elsewhere?

Reply:
It is (['AI2020', 'DocDef2', 'W00']) everywhere now. This is because the datasets are named that, and those names carry over to the model. But I agree that scholarphi should not have anything to do with the dataset names.
How should we handle this going forward @dykang ?

Reply:
Good point. There are some inquiries about our HEDDEx model, and I pointed them to see codes in this repo for now. In that regard, I believe we should keep the dataset names clear like as they are. But, I agree that they should be in a sort of configuration file in the future.